### ISSUE TITLE
why the 'get_direction_dir_sign' function returns 'dot_prod<0' instead 'dot_prod>0' ?? 

### ISSUE THREAD

**Issue by hailey94** [tensorflow/tcav#107]
Hi, i'm curious about compute tcav score in this code.
As shown in the paper, calculation of TCAV is the ratio of inputs whose conceptual sensitivities are (+) sign (> 0). But why this source code counts when the conceptual  sensitivities are < 0 ??  Or, am I misunderstanding a paper or code??

![tcav](https://user-images.githubusercontent.com/29304469/113009617-11e00680-91b3-11eb-927e-2297793fcd8d.JPG)

**Comment by BeenKim** [comment]
Hi, Your understanding is correct!
The source code is taking derivative wrt the loss instead of p(x) (logit) so the sign flips. The lower the loss is the more likely the p(x) is. Hope this helps!
